hotsos/core/host_helpers/uptime.py

```diff
@@ -58,14 +64,22 @@ def minutes(self):
         log.warning("unknown uptime format in %s", self.uptime)
 
     @property
-    def seconds(self):
-        if self.minutes:
-            return self.minutes * 60
+    def in_seconds(self):
+        """ Total uptime in seconds. """
+        if self.in_minutes:
+            return self.in_minutes * 60
 
     @property
-    def hours(self):
-        if self.minutes:
-            return int(self.minutes / 60)
+    def in_hours(self):
+        """ Total uptime in hours. """
+        if self.in_minutes:
+            return int(self.in_minutes / 60)
+
+    def __repr__(self):
+        days = int(self.in_hours / 24)
+        hours = self.in_hours - days * 24
+        minutes = self.in_minutes - (self.in_hours * 60)
```

Comment:
It looks like all in_* methods can return None if their respective in_* values are zero. Should they have a 'return 0' as last line to force a valid value? Else some of these calculations will throw a TypeError I think.

Reply:
fwict this is ok now